src/components/Select/Select.scss

```diff
@@ -0,0 +1,96 @@
+@import '~@quartz/styles/scss/borders';
+@import '~@quartz/styles/scss/color-scheme';
+@import '~@quartz/styles/scss/fonts';
+@import '~@quartz/styles/scss/helpers/positioning';
+@import '~@quartz/styles/scss/helpers/resets';
+
+.container {
+	@include font-maison;
+}
+
+.select-container {
+	position: relative;
+}
+
+.select {
+	@include reset-select;
+
+	display: inline-block;
+	height: 45px;
+	width: 100%;
+	min-width: 210px;
+	border-radius: $size-border-radius;
+	border: $border-solid-interactive;
+	cursor: pointer;
+	font-size: 16px;
+	color: $color-typography;
+	padding: 0 15px 0 40px;
+	background-color: $color-background-2;
+
+	&:disabled {
+		opacity: 0.5;
+
+		&:hover {
+			border-color: $color-border-interactive;
+		}
+	}
+
+	&[aria-invalid="true"] {
+		color: $color-red;
+		border-color: $color-red;
+	}
+
+	&.is-placeholder {
+		color: $color-typography-faint;
+	}
+
+	&:hover,
+	&:focus {
+		border-color: $color-accent;
+	}
+
+	&::-ms-expand {
+		display: none;
+	}
```

Comment:
I'd be inclined to leave this out for a couple of reasons: 
1) I don't think we should take it upon ourselves to clean up after browser vendors' (misguided) attempts to style form elements in non-standard ways, and users probably don't expect us to do so either
2) I suspect this only applies to IE and old Edge browsers which have fallen off our support map
```suggestion
```